content/01_vectors.html

```diff
@@ -915,7 +927,7 @@ <h3 id="example-19-motion-101-velocity-and-random-acceleration">Example 1.9: Mot
 <p>You might also notice that this example is another kind of random walker.<strong><em> </em></strong>A key distinction between what I’m doing here and the previous chapter’s examples, however, lies in <em>what</em> is being randomized. With the traditional random walker, I was directly manipulating the velocity, meaning each step was completely independent of the last. In Example 1.9, the acceleration (the rate of change of velocity) is being randomized, not the velocity itself. This makes the object’s motion dependent on its previous state: the velocity changes incrementally according to the random acceleration. The resulting movement of the object has a kind of continuity and fluidity that the original random walker lacked. The difference may seem subtle, but it fundamentally changes the way the object moves about the canvas.</p>
 <div data-type="exercise">
   <h3 id="exercise-16">Exercise 1.6</h3>
-  <p>Referring back to Example 0.6, implement an acceleration calculated with Perlin noise.</p>
+  <p>Referring back to Example 0.7, implement an acceleration calculated with Perlin noise.</p>
```

Comment:
```suggestion
  <p>Referring back to Example 0.6, implement an acceleration calculated with Perlin noise.</p>
```
whoops, let's revert this